[2024-01-27T04:43:09.978+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_api_dag.get_weather_data scheduled__2024-01-27T00:00:00+00:00 [queued]>
[2024-01-27T04:43:09.998+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_api_dag.get_weather_data scheduled__2024-01-27T00:00:00+00:00 [queued]>
[2024-01-27T04:43:09.998+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-01-27T04:43:10.049+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): get_weather_data> on 2024-01-27 00:00:00+00:00
[2024-01-27T04:43:10.059+0000] {standard_task_runner.py:60} INFO - Started process 4325 to run task
[2024-01-27T04:43:10.071+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_api_dag', 'get_weather_data', 'scheduled__2024-01-27T00:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/weather_api__dag.py', '--cfg-path', '/tmp/tmp_xifykko']
[2024-01-27T04:43:10.090+0000] {standard_task_runner.py:88} INFO - Job 8: Subtask get_weather_data
[2024-01-27T04:43:10.210+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_api_dag.get_weather_data scheduled__2024-01-27T00:00:00+00:00 [running]> on host d24a8cdd2842
[2024-01-27T04:43:10.378+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_api_dag' AIRFLOW_CTX_TASK_ID='get_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-27T00:00:00+00:00'
[2024-01-27T04:43:10.379+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-27T04:43:10.392+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_api_dag, task_id=get_weather_data, execution_date=20240127T000000, start_date=20240127T044309, end_date=20240127T044310
[2024-01-27T04:43:10.453+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-27T04:43:10.493+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
